test/e2e/tests/interpreters/new-uv-project.test.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import path from 'path';
+import { test, tags } from '../_test.setup';
+
+test.use({
+	suiteId: __filename
+});
+
+test.describe('New UV Environment', {
+	tag: [tags.INTERPRETER, tags.WEB]
+}, () => {
+
+	test.beforeAll(async function ({ userSettings }) {
+		await userSettings.set([['console.multipleConsoleSessions', 'true']], true);
+	});
+
+	test('Python - Add new UV environment', async function ({ app, openFolder }) {
+
+		await app.workbench.terminal.clickTerminalTab();
+
+		await app.workbench.terminal.runCommandInTerminal('uv init proj');
```

Comment:
I think this is the way to ask for a specific version so the last test checking version will pass 🙂 
```suggestion
		await app.workbench.terminal.runCommandInTerminal('uv init proj -p 3.13.1');
```

Reply:
Oh, good idea.  I was just changing the test to do this instead:

		const metadata = await app.workbench.sessions.getMetadata();

		expect(metadata.source).toBe('Venv');
		expect(metadata.path).toBe('/tmp/vscsmoke/qa-example-content/proj/.venv/bin/python');
		
so a specific version check is not needed.  I will fall back on the specific version if necessary.